core/src/main/scala/org/genivi/sota/core/rvi/TransferController.scala

```diff
@@ -161,20 +161,18 @@ class TransferProtocolActor(db: Database, rviClient: RviClient, transferActorPro
     case x @ ChunksReceived(vin, p, _) =>
       inProgress.find( _._2.id == p).foreach( _._1 ! x)
 
-    case r @ InstallReport(vin, packageId, success, msg) =>
+    case r @ InstallReport(vin, update) =>
```

Comment:
@jerrytrieu please have a thorough look if that is what we want; also please check on how to decide if success